File: Montes-Guerrero-Daniel/practica2/procesos.py
"""
Alumno: Daniel Montes Guerrero
Ejecutar: python procesos.py
"""
import os

pid = os.fork()

if pid > 0:
    print("Soy el padre", os.getpid())
else:
    print("Soy el hijo", os.getpid(), "con padre", os.getppid())

File: Montes-Guerrero-Daniel/practica2/Procesos.java
/*
 * Alumno: Daniel Montes Guerrero
 * Compilar: javac Procesos.java && java Procesos
 */

class ThreadExample extends Thread {
	public void run(){
		System.out.println("Soy el hijo " + ProcessHandle.current().pid());
	}
}

public class Procesos {
	public static void main(String[] args){
		ThreadExample thread = new ThreadExample();
		thread.start();
		System.out.println("Soy el padre " + ProcessHandle.current().pid());
	}
}
